_data/services.yml

```diff
@@ -1,3 +1,19 @@
+sync:
+  name: Data Sync
+  icon: sync
+  link: /services/data-sync/
+  description: Add realtime data synchronization and offline capabilities to your mobile app
+  features:
+    - Use GraphQL to design the APIs for your mobile app
+    - Add real time data synchronisation to your mobile app
+    - Allow your moible app to work offline
```

Comment:
Typo. I think mobile app sounds too vague and may be confusing for native felas. If target is cordova and web then we should specify that. 

The main target for this is to give plain facts about sync.

- Works with web and cordova
- Uses GraphQL (that will imply many features out of the box)

Reply:
the sdks sections clearly indicates what sdks are available. For this section, I think a bit high level is ok.